scripts/Tools/Makefile

```diff
@@ -139,7 +139,7 @@ ifeq ($(strip $(PIO_VERSION)),1)
 endif
 
 ifeq (,$(SHAREDPATH))
-  SHAREDPATH = $(COMPILER)/$(MPILIB)/$(DEBUGDIR)/$(THREADDIR)
+  SHAREDPATH = $(COMPILER)/$(MPILIB)/$(DEBUGDIR)/$(THREADDIR)/$(COMP_INTERFACE)
```

Comment:
This is needed because of the CPP def in perf_mod .  (otherwise in the test suite one setting overwrites the other)